### PR TITLE
Uperf single run

### PR DESCRIPTION
**Pull request by whitleykeith**
### Description

Current Uperf Scripts for pod and service network tests actually run three separate runs per script invocation (different number of pairs). In Airflow we assume 1 benchmark per task, so we want to run those three in separate tasks. 

This script `run.sh` will just run any generic uperf workload, and take in the necessary params to configure it. We can use this in our Airflow benchmark configs to split out the runs into separate tasks.

### CODE REVIEW

File: workloads/network-perf/run.sh [hunk 1]
```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+
+source ./common.sh
+export WORKLOAD=${WORKLOAD}
+
+if [[ $WORKLOAD == "hostnet" ]]; then
+    export HOSTNETWORK=true
+elif [[ $WORKLOAD == "service" ]]; then
+    export SERVICEIP=true 
+fi
```
Comment (Harshith-umesh) [inline workloads/network-perf/run.sh:10]: else if $WORKLOAD == "pod" condition missing for pod-network-test

Comment (whitleykeith) [reply workloads/network-perf/run.sh:10]: what gets set from that conditional? the other workload vars changed another env but i don't see a var to change

Comment (Harshith-umesh) [reply workloads/network-perf/run.sh:10]: right

**Comment by Harshith-umesh** [comment]
LGTM

**Review by mohit-sheth — APPROVED** [review]

**Pull request merged**